cypress/integration/Toggle.spec.js

```diff
@@ -0,0 +1,79 @@
+/// <reference types="Cypress" />
+
+/* eslint-disable no-undef */
+describe('The Toggle component', () => {
+  beforeEach(() => {
+    cy.visit('/iframe.html?selectedKind=Action%7CToggle&selectedStory=basic%20toggle&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel');
+  });
+
+  it('Toggles!', () => {
+    cy.get('label').click();
+    cy.get('label > input').should('be.checked');
+    cy.get('label').click();
+    cy.get('label > input').should('not.be.checked');
+    cy.get('label').click();
+    cy.get('label > input').should('be.checked');
+    cy.get('label > input').focus();
+  });
+
+//   it('Focus toggles!', () => {
+//     cy.get('label > input').focus(); // need to do space bar toggle then check
```

Comment:
@nicklemmon - I was trying `keypress` events using `trigger()` as well as `type()` calls but can't get the spacebar event to toggle the checkbox value through Cypress.

Reply:
@nicklemmon <3